tests/full_node_test.cpp

```diff
@@ -906,7 +906,7 @@ TEST_F(FullNodeTest, reconstruct_dag) {
     // TODO: pbft does not support node stop yet, to be fixed ...
     node->getPbftManager()->stop();
     for (size_t i = 1; i < num_blks; i++) {
-      node->getDagManager()->addDagBlock(g_mock_dag0[i]);
+      node->getDagManager()->addDagBlock(std::move(mock_dags[i]));
```

Comment:
now you are moving local variable 2x

Reply:
bad news is, second iteration of clang-tidy analysis did not find it...